### ISSUE TITLE
Donkeytown low pop state needs to be calculated separately from server pop

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLife#1133]
Donkeytown is basically constantly in a state of low population yet the biomes are always locked (jungle, desert, arctic)

The lowpop state for donkeytown needs to be calculated separately from the server lowpop state.

**Comment by DougLefelhocz** [comment]
Arguably, at no point should a donkeytown player fly to the main area.

However, donkeytown players (and tutorial players) is and has been included in the number of players for race restriction threshold.  I don't know if that works as intentional or not though.  But, it does seem abuse-able in principle that players with multiple accounts could raise the number of players to over 40, when say there exist only 20 players in the main area causing an issue with main area players able to get resources from race restricted biomes.  Tutorial players also.

**Comment by Rukhmar** [comment]
We have so many small servers that are hardly ever used, why not repurpose one of them into a DT server that it takes them to if they are cursed out of the main server (ignoring their preference if they set it to BS2 and going there anyways)? Then it can have its own families and lowpop rules, not connected to the main server. Also we won't have to worry as much about exploits flying people in and out of DT if they are on a completely different server.

**Comment by Forman9** [comment]
My point is you can't get into the jungle to get rubber or the arctic to get oil except when the WHOLE SERVER is on low pop
(so that you can enter any biome)

DT should ALWAYS be on low pop cause there are never more than 5-10 people there.

**Comment by DougLefelhocz** [comment]
> We have so many small servers that are hardly ever used

Small server is a player invented term.  I use it, and it's fine within it's context.

However, I think Jason's perspective those are more like overflow servers.  The original design was for all of them to get used eventually by people not checking a custom server.  So, like let's say server1 got used for DT players.  But then there's a sale.  And now server1 becomes overflow and used by players not checking a custom server.  Where do the new DT players go?  Alright, so maybe you say server2.  But, suppose there's a bigger sale.  Now, DT players have to go to server 3.  And then we have a recursive problem, where if *all* servers got filled, purchasing an extra server would need done to suit DT players.

Also, if DT players were in a main area on s1, and there was a sale, then all the current DT players on s1 would get new players babies!  Who knows how many would get fed to bears, even as a baby?  I can't imagine getting fed to a bear as a positive experience for a new player when playing with a random person.  It can be funny as a joke with friends or people I've talked to on stream.  But, with some random person it's not just offensive, but concerning that some random person did that (though maybe they would think I was a bot, not a human being).

Sure, maybe DT players do something else instead of feeding them to bears.  But, it's probably still something bad for new players especially.

**Comment by DougLefelhocz** [comment]
> My point is you can't get into the jungle to get rubber or the arctic to get oil except when the WHOLE SERVER is on low pop (so that you can enter any biome)
> 
> DT should ALWAYS be on low pop cause there are never more than 5-10 people there.

Low pop is and has always been server based.  Tutorial players and donkeytown players count towards whether a server is high pop or not.

Do you think that it's an issue that a server can be high pop when there exist 6-8 DT and tutorial players total, and the server only has 41 people on it?

**Comment by Rukhmar** [comment]
I never said it had to be server 1, since it is used when there is maintenance or a streamer wave. It could be 13, since the way it is going, we won't ever need that much overflow. He intended for them to all be overflow servers, but you can maybe use 2 or 3 tops even in the big spikes, so most sit either empty or with 1-2 people building a small base. Just repurposing one for DT wouldn't make much impact in that, and might help DT not be as exploited on the main server. Plus they would be low pop so they can go into any biome and build anything like they have been requesting for a while.

**Comment by jasonrohrer** [comment]
Yeah, d-town is what it is.  You gotta deal with what you gotta deal with out there.  It's the Australian Outback (prison colony).  It's not supposed to have bidets and caviar.
